clang/test/CXX/drs/dr20xx.cpp

```diff
@@ -61,6 +61,14 @@ namespace dr2026 { // dr2026: 11
   }
 }
 
+namespace dr2049 { // dr2049: 18 drafting
+#if __cplusplus > 202002L
+template <int* x = {}> struct X {};
+X<> a;
+X<nullptr> b;
```

Comment:
Perhaps add a same-type static assertion for `a` and `b` to ensure a consistent specialization is selected?